marklogic-data-hub-central/ui/e2e/cypress/support/pages/browse.tsx

```diff
@@ -185,7 +186,7 @@ class BrowsePage {
   }
 
   getHubPropertiesExpanded() {
-    return cy.get("#hub-properties > div").eq(1).invoke('show').click();
```

Comment:
@bsrikan I remember adding this line fixed some of the scenarios.json and xml e2e tests because some times it's hard to find the expand icon while the page is still loading and the test fails on click, that's the reason we thought of clicking on hub properties text. Any reason for this change ?

Reply:
Added info: I could have changed it to use data-testid="toggle", but there is a duplicate data-testid for entity properties, which should be fixed, since locator id should be unique.